### PR TITLE
Update benchmark and control tags for compatibility with dashboard page Closes #9

### LINKED ISSUE #9: Update benchmark and control tags for compatibility with dashboard page
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by rajlearner17**
### Checklist
- [ ] Issue(s) linked
screencapture-localhost-9195-2022-05-05-16_26_50
screencapture-localhost-9195-2022-05-05-16_27_06
screencapture-hub-steampipe-io-git-staging-turbot-vercel-app-mods-turbot-zoom-compliance-controls-control-cis-v100-1-1-1-1-1-2022-05-05-16_28_17
screencapture-hub-steampipe-io-git-staging-turbot-vercel-app-mods-turbot-zoom-compliance-controls-control-cis-v100-1-3-1-2022-05-05-16_29_05

### CODE REVIEW

File: cis_v100/section_1_1_2.sp [hunk 1]
```diff
@@ -1,9 +1,12 @@
 
-benchmark "cis_v100_1_1_2" {
-  title         = "1.1.2 Schedule Meeting"
-  tags = merge(local.cis_v100_common_tags, {
-    cis_item_id = "1.1.2"
+locals {
+  cis_v100_1_1_2_common_tags = merge(local.cis_v100_common_tags, {
```
Comment (khushboo9024) [inline cis_v100/section_1_1_2.sp]:
```suggestion
  cis_v100_1_1_2_common_tags = merge(local.cis_v100_1_1_common_tags, {
```

File: cis_v100/section_1_1_3.sp [hunk 2]
```diff
@@ -1,9 +1,11 @@
+locals {
+  cis_v100_1_1_3_common_tags = merge(local.cis_v100_common_tags, {
```
Comment (khushboo9024) [inline cis_v100/section_1_1_3.sp:2]:
```suggestion
  cis_v100_1_1_3_common_tags = merge(local.cis_v100_1_1_common_tags, {
```

File: cis_v100/section_1_1_4.sp [hunk 3]
```diff
@@ -1,9 +1,12 @@
 
-benchmark "cis_v100_1_1_4" {
-  title         = "1.1.4 In Meeting (Advanced)"
-  tags = merge(local.cis_v100_common_tags, {
-    cis_item_id = "1.1.4"
+locals {
+  cis_v100_1_1_4_common_tags = merge(local.cis_v100_common_tags, {
```
Comment (khushboo9024) [inline cis_v100/section_1_1_4.sp:3]:
```suggestion
  cis_v100_1_1_4_common_tags = merge(local.cis_v100_1_1_common_tags, {
```

File: cis_v100/section_3.sp [hunk 4]
```diff
@@ -1,56 +1,93 @@
...
+
+locals {
+  cis_v100_3_1_1_1_common_tags = merge(local.cis_v100_3_1_1_common_tags, {
+    cis_section_id = "3.1.1.1"
+  })
+  cis_v100_3_1_1_2_common_tags = merge(local.cis_v100_3_1_1_common_tags, {
+    cis_section_id = "3.1.1.2"
   })
+  cis_v100_3_1_1_3_common_tags = merge(local.cis_v100_3_1_1_common_tags, {
+    cis_section_id = "3.1.1.2"
```
Comment (khushboo9024) [inline cis_v100/section_3.sp:27]:
```suggestion
    cis_section_id = "3.1.1.3"
```

File: cis_v100/section_1_1_7.sp [hunk 5]
```diff
@@ -1,57 +1,65 @@
 
-benchmark "cis_v100_1_1_7" {
-  title         = "1.1.7 Admin Options"
-  tags = merge(local.cis_v100_common_tags, {
-    cis_item_id = "1.1.7"
+locals {
+  cis_v100_1_1_7_common_tags = merge(local.cis_v100_common_tags, {
```
Comment (khushboo9024) [inline cis_v100/section_1_1_7.sp:3]:
```suggestion
  cis_v100_1_1_7_common_tags = merge(local.cis_v100_1_1_common_tags, {
```

File: cis_v100/section_1_1_7.sp [hunk 6]
```diff
@@ -1,57 +1,65 @@
...
+benchmark "cis_v100_1_1_7" {
+  title = "1.1.7 Admin Options"
   children = [
     control.cis_v100_1_1_7_1,
     control.cis_v100_1_1_7_2,
     control.cis_v100_1_1_7_3,
     control.cis_v100_1_1_7_4,
   ]
+
+  tags = merge(local.cis_v100_1_1_6_common_tags, {
```
Comment (khushboo9024) [inline cis_v100/section_1_1_7.sp:17]:
```suggestion
  tags = merge(local.cis_v100_1_1_7_common_tags, {
```

File: cis_v100/section_1_1_6.sp [hunk 7]
```diff
@@ -12,27 +15,34 @@ benchmark "cis_v100_1_1_6" {
...
-    cis_item_id = "1.1.6.1"
-  })
+  title = "1.1.6.1 When a cloud recording is available"
   children = [
     control.cis_v100_1_1_6_1_1,
     control.cis_v100_1_1_6_1_2,
     control.cis_v100_1_1_6_1_3,
   ]
+
+  tags = merge(local.cis_v100_1_1_6_common_tags, {
```
Comment (khushboo9024) [inline cis_v100/section_1_1_6.sp:33]: We should define another tags `cis_v100_1_1_6_1_common_tags`

File: cis_v100/section_1_1_5.sp [hunk 8]
```diff
@@ -1,21 +1,29 @@
 
-benchmark "cis_v100_1_1_5" {
-  title         = "1.1.5 Calendar and Contacts"
-  tags = merge(local.cis_v100_common_tags, {
-    cis_item_id = "1.1.5"
+locals {
+  cis_v100_1_1_5_common_tags = merge(local.cis_v100_common_tags, {
```
Comment (khushboo9024) [inline cis_v100/section_1_1_5.sp:3]:
```suggestion
  cis_v100_1_1_5_common_tags = merge(local.cis_v100_1_1_common_tags, {
```

File: cis_v100/section_2.sp [hunk 9]
```diff
@@ -1,268 +1,307 @@
...
   children = [
     benchmark.cis_v100_2_1_1,
     benchmark.cis_v100_2_1_2,
     benchmark.cis_v100_2_1_3,
     benchmark.cis_v100_2_1_4,
   ]
+
+  tags = merge(local.cis_v100_2_1_common_tags, {
+    service = "Zoom/IM"
+    type = "Benchmark"
```
Comment (cbruno10) [inline cis_v100/section_2.sp]:
```suggestion
    service = "Zoom/IM"
    type    = "Benchmark"
```

File: cis_v100/section_2.sp [hunk 10]
```diff
@@ -1,268 +1,307 @@
...
+  })
 }
 
 control "cis_v100_2_1_1_1" {
   title       = "2.1.1.1 Ensure screen capture is set to disabled (Manual)"
   description = "Allow users to take and send screenshots in direct messages or group conversations. Reduces likelihood of data leakage through zoom screenshot sharing."
-  sql = query.no_api_available.sql
-  tags = merge(local.cis_v100_common_tags, {
+  sql         = query.no_api_available.sql
+  tags = merge(local.cis_v100_2_1_1_common_tags, {
```
Comment (cbruno10) [inline cis_v100/section_2.sp:76]: Should there be service tags here on the controls as well (and the other controls as well)?

File: cis_v100/section_1_1_2.sp [hunk 11]
```diff
@@ -34,12 +39,16 @@ benchmark "cis_v100_1_1_2_1" {
     control.cis_v100_1_1_2_1_5,
     control.cis_v100_1_1_2_1_6,
   ]
+
+  tags = merge(local.cis_v100_1_1_2_common_tags, {
+    type        = "Benchmark"
```
Comment (cbruno10) [inline cis_v100/section_1_1_2.sp:50]:
```suggestion
    type = "Benchmark"
```

File: cis_v100/section_1_1_4.sp [hunk 12]
```diff
@@ -23,23 +26,30 @@ benchmark "cis_v100_1_1_4" {
...
-    cis_item_id = "1.1.4.1"
-  })
+  title = "1.1.4.1 Select data center regions for meetings/webinars hosted by your account"
   children = [
     control.cis_v100_1_1_4_1_1,
     control.cis_v100_1_1_4_1_2,
   ]
+
+  tags = merge(local.cis_v100_1_1_4_common_tags, {
+    service     = "Zoom/Meeting"
```
Comment (cbruno10) [inline cis_v100/section_1_1_4.sp:59]: Spacing looks off here?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@rajlearner17 Overall suggestions:
- Please check spacing on all key/value pairs and ensure they're aligned
- Do controls need `service` tags? I assume yes, but wanted to check if there's a reason we decided not to include them.

Also, do you have updated screenshots you can replace the original ones with in this PR? Thanks!

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**